### PR TITLE
ROC-3067 Defendant content updates

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3067

### Change description
Updated several pages of the defendant journey to match prototype content. 

### Work checklist

- [x] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/first-contact/routes/claim-summary.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { ClaimReferenceMatchesGuard } from 'first-contact/guards/claimReferenceM
 import { JwtExtractor } from 'idam/jwtExtractor'
 import { ClaimantRequestedCCJGuard } from 'first-contact/guards/claimantRequestedCCJGuard'
 import { OAuthHelper } from 'idam/oAuthHelper'
+import { getInterestDetails } from '../../../common/interest'
```
Comment (rlewan) [inline src/main/features/first-contact/routes/claim-summary.ts]: Please change this to an absolute import.

File: src/main/features/claim/helpers/feesTableViewHelper.ts [hunk 2]
```diff
@@ -1,4 +1,9 @@
 import { isUndefined } from 'util'
+import { FeesClient } from 'fees/feesClient'
+import { FeeRange as MergeableRange } from 'claim/helpers/feesTableViewHelper'
+import { FeeRange as ViewFeeRange } from 'fees/models/feeRange'
+
+const supportedFeeLimitInGBP: number = 10000
```
Comment (timja-kainos) [inline src/main/features/claim/helpers/feesTableViewHelper.ts]: please tell me this is the only place this is in?

File: src/main/features/response/views/more-time-request.njk [hunk 3]
```diff
@@ -36,7 +35,8 @@
                 bold: true
               }
             ],
-            form = form
+            form = form,
+            legendHidden = true
```
Comment (timja-kainos) [inline src/main/features/response/views/more-time-request.njk]: is the legend duplicating the label?

File: src/main/features/first-contact/routes/claim-summary.ts (hunk 1 above)
Comment (robertparkinson) [reply src/main/features/first-contact/routes/claim-summary.ts]: done, intelliJ auto imports it using all the dots. Need to train myself to check for it in future.

File: src/main/features/response/views/your-dob.njk [hunk 4]
```diff
@@ -3,17 +3,18 @@
...
   <div class="grid-row">
     <div class="column-two-thirds">
+      <p>{{ t('We use your date of birth to make sure you’re over 18 and that legal proceedings involving a claim are taken against the right person.') }}</p>
       <form method="post">
         <input type="hidden" name="known" value="true" />
 
         {{ csrfProtection(csrf) }}
 
-        {{ dateInput(form = form, name = 'date', hint = 'You must be over 18 to use this service') }}
+        {{ dateInput(form = form, name = 'date', hint = 'For example, 31 3 1980') }}
```
Comment (timja-kainos) [inline src/main/features/response/views/your-dob.njk]: is that the default hint?

File: src/main/features/response/views/counter-claim.njk [hunk 5]
```diff
@@ -1,47 +1,39 @@
...
 
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
       <div class="panel-indent">
         <div class="panel panel-border-wide">
-          <p>{{ t('Complete and email the defence and counterclaim form by %s',
-            { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</p>
-          <p>{{ t('To make a counter claim you will have to pay a fee. Court staff can tell you how much you have to pay.') }}</p>
+          <p>{{ t('Complete and email the defence and counterclaim form by') }} <span class="bold-small">{{ claim.responseDeadline | date }}</span></p>
```
Comment (rlewan) [inline src/main/features/response/views/counter-claim.njk]: I think the whole content of `<p>` should be translated and filter through `safe` like it's done in other places.

File: src/main/features/claim/helpers/feesTableViewHelper.ts (hunk 2 above)
Comment (robertparkinson) [reply src/main/features/claim/helpers/feesTableViewHelper.ts]: yup. moved it there as it is now needed twice

File: src/main/features/response/views/your-dob.njk [hunk 6]
```diff
@@ -3,17 +3,18 @@
 {% from "date.njk" import dateInput %}
 
 {% set title = t('Money claim') %}
-{% set heading = t('What is your date of birth?') %}
+{% set heading = t('Add your date of birth') %}
 
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
+      <p>{{ t('We use your date of birth to make sure you’re over 18 and that legal proceedings involving a claim are taken against the right person.') }}</p>
```
Comment (timja-kainos) [inline src/main/features/response/views/your-dob.njk]: could you break this into multiple lines please?
less scrolling is better for re

File: src/main/features/response/views/counter-claim.njk [hunk 7]
```diff
@@ -1,47 +1,39 @@
...
           <ul class="list list-bullet">
             <li>{{ t('Claim No.') }} {{ claim.claimNumber }}</li>
             <li>{{ t('Claimant') }} {{ claim.claimData.claimant.name }}</li>
             <li>{{ t('Defendant') }} {{ response.defendantDetails.partyDetails.name }}</li>
           </ul>
         </li>
-        <li>{{ t('Download and attach the completed form to an email') }}</li>
-        <li>{{ t('Send the email with the attached form to') }} <a href="mailto:{{ supportEmailAddress }}?Subject={{ claim.claimNumber }}">{{ supportEmailAddress }}</a></li>
+        <li>{{ t('Save the completed form to your device.') }}</li>
+        <li>{{ t('Email the form to') }} <a href="mailto:{{ supportEmailAddress }}?Subject={{ claim.claimNumber }}">{{ supportEmailAddress }}</a></li>
```
Comment (rlewan) [inline src/main/features/response/views/counter-claim.njk]: Similar case here, whole `<li>` content should be translated.

File: src/main/features/response/views/more-time-request.njk (hunk 3 above)
Comment (robertparkinson) [reply src/main/features/response/views/more-time-request.njk]: have updated to make the label match the title

File: src/main/views/macro/feesTable.njk [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="panel">
+      <p>
+        {{ t('The fee is based on the amount you’re claiming, including any interest.')}}
+      </p>
+      <table>
+        <thead>
+        <tr>
+          <th scope="col">{{ t('Claim amount') }}</th>
+          <th scope="col">{{ t('Claim fee') }}</th>
+          {% if noHearingFee %}<th scope="col">{{ t('Hearing fee') }}</th>{% endif %}
```
Comment (timja-kainos) [inline src/main/views/macro/feesTable.njk]: err, if no hearing fee then add hearing fee?

File: src/main/features/response/views/more-time-confirmation.njk [hunk 9]
```diff
@@ -10,11 +10,11 @@
     <div class="column-two-thirds">
 
       <p>
-        {{ t('Your claim response deadline is now %s.', { postProcess: 'sprintf', sprintf: [newDeadline | date] }) }}
+        {{ t('You must respond before {{ deadline }}.', { deadline: newDeadline | date }) }}
```
Comment (rlewan) [inline src/main/features/response/views/more-time-confirmation.njk]: Here as well.

File: src/main/views/macro/feesTable.njk (hunk 8 above)
Comment (timja-kainos) [reply src/main/views/macro/feesTable.njk]: how about `includeHearingFee` and then default it to true above? not sure which option is more common though

File: src/main/features/response/views/your-dob.njk (hunk 4 above)
Comment (robertparkinson) [reply src/main/features/response/views/your-dob.njk]: true! removed.

File: src/main/features/response/views/counter-claim.njk (hunk 5 above)
Comment (robertparkinson) [reply src/main/features/response/views/counter-claim.njk]: done

File: src/main/features/response/views/your-dob.njk (hunk 6 above)
Comment (robertparkinson) [reply src/main/features/response/views/your-dob.njk]: done

File: src/main/features/response/views/counter-claim.njk (hunk 7 above)
Comment (robertparkinson) [reply src/main/features/response/views/counter-claim.njk]: done

File: src/main/views/macro/feesTable.njk (hunk 8 above)
Comment (robertparkinson) [reply src/main/views/macro/feesTable.njk]: updated to use showHearingFee

Comment (robertparkinson) [reply src/main/views/macro/feesTable.njk]: it's only used in two places so it's either/or at the moment although I expect showing just the claim fees will be more than hearing centre fees

File: src/main/features/claim/views/fees.njk [hunk 10]
```diff
@@ -37,32 +38,7 @@
...
-                <tr>
-                  <td>{{ row.minRange | numeral }} to {{ row.maxRange | numeral }}</td>
-                  <td>{{ row.fees[1] | numeral | default('Not available') }}</td>
-                  <td>{{ row.fees[2] | numeral | default('Not available') }}</td>
-                </tr>
-              {% endfor %}
-            </table>
-          </div>
-        </details>
+        {{ feesTable('How these fees are calculated', rows ) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/fees.njk]: shouldn't this one show the heading fe?

File: src/main/features/response/views/counter-claim.njk [hunk 11]
```diff
@@ -1,47 +1,40 @@
...
-          <p>{{ t('Complete and email the defence and counterclaim form by %s',
-            { postProcess: 'sprintf', sprintf: [claim.responseDeadline | date]}) }}</p>
-          <p>{{ t('To make a counter claim you will have to pay a fee. Court staff can tell you how much you have to pay.') }}</p>
+          <p>{{ t('Complete and email the defence and counterclaim form by <span class="bold-small">{{ responseDeadline }}', { responseDeadline: claim.responseDeadline | date  }) | safe }} </span></p>
         </div>
       </div>
 
+      <p>{{ t('You’ll need to pay a court fee if you make a counterclaim.') }}</p>
+
+      {{ feesTable('View claim fees', rows, showHearingFee = true ) }}
```
Comment (timja-kainos) [inline src/main/features/response/views/counter-claim.njk]: shouldn't this one hide the hearing fee?

File: src/main/features/response/views/counter-claim.njk [hunk 12]
```diff
@@ -1,47 +1,40 @@
...
+
       <ul class="list list-number">
         <li>
-          {{ t('Complete <a href="%s" target="_blank">defence and counterclaim form N9B</a> <span class="font-xsmall">(opens in a new window)</span> entering these details:',
-          { postProcess: 'sprintf', sprintf: [asset_paths["pdf"] + '/OCON9B.pdf']}) | safe }}
+          {{ t('Complete <a href="{{ link }}" download="download" target="_blank">defence and counterclaim form N9B</a> entering these details:',
```
Comment (timja-kainos) [inline src/main/features/response/views/counter-claim.njk]: the download attribute doesn't work in IE or iOS safari, probably fine but something to be aware of

File: src/main/features/response/views/more-time-request.njk (hunk 3 above)
Comment (timja-kainos) [reply src/main/features/response/views/more-time-request.njk]: FYI for future when we have similar cases:
> Ed Horsford [10 hours ago]
We now recommend nesting the h1 within the legend, so you don’t get anything repeated and everyone is happy.

File: src/main/features/response/views/counter-claim.njk (hunk 11 above)
Comment (robertparkinson) [reply src/main/features/response/views/counter-claim.njk]: you're right, removed. although it looks like that page has now changed again :/

File: src/main/features/response/views/eligibility/send-your-response-by-email.njk [hunk 13]
```diff
@@ -3,97 +3,109 @@
...
         </li>
       </ul>
 
-      {{ supportWidget() }}
-      {% include "includes/service_survey_link.njk" %}
+      {% if draft.response.type === ResponseType.DEFENCE %}
+        {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.COUNTER_CLAIM %}
 
-    </div>
+          <h2 class="heading-medium">{{ t('Counterclaim fee') }}</h2>
```
Comment (robertparkinson) [inline src/main/features/response/views/eligibility/send-your-response-by-email.njk]: Please use the feesTable macro I've created: 
 {{ feesTable('View claim fees', rows ) }}

File: src/main/features/response/views/eligibility/send-your-response-by-email.njk [hunk 14]
```diff
@@ -78,35 +67,62 @@
...
-          <p>{{ t(externalLink('Download defence form', asset_paths["pdf"] + '/OCON9B.pdf') + ' (PDF, 75 KB).') | safe }}</p>
+        {% if draft.response.type === ResponseType.FULL_ADMISSION %}
+          <li>{{ t('Print and complete the form') }}</li>
         {% endif %}
-      {% endif %}
+
+        <li>{{ t('Post the forms to:') }}</li>
+
+        <ul class="panel">
+          <li>{{ t('Civil Money Claims') }}</li>
```
Comment (robertparkinson) [inline src/main/features/response/views/eligibility/send-your-response-by-email.njk]: don't use a ul/li for this please use something like this: 
```<div class="keep-formatting">County Court Money Claims Centre
    PO Box 527
    Salford
    Greater Manchester
    M5 0BY
    DX: 702634 Salford 5</div>```

**Pull request merged**